include/logging/log_core.h

```diff
@@ -407,6 +407,14 @@ static inline char z_log_minimal_level_to_char(int level)
 extern struct log_source_const_data __log_const_start[0];
 extern struct log_source_const_data __log_const_end[0];
 
+/**@brief Enum with possible actions for strdup operation.
+ */
+enum _strdup_action {
```

Comment:
`log_` prefix missing. File is included by log.h so by everything. Apart from that, zephyr avoids names starting with `_`